src/dispatchers/touchDispatcher.ts

```diff
@@ -118,12 +118,19 @@ export module Dispatcher {
      * calls broadcast() on the supplied Broadcaster.
      */
     private _measureAndBroadcast(e: TouchEvent, b: Core.Broadcaster<Dispatcher.Touch>) {
-      var touch = e.changedTouches[0];
-      var newTouchPosition = this.translator.computePosition(touch.clientX, touch.clientY);
-      if (newTouchPosition != null) {
-        this._lastTouchPosition = newTouchPosition;
-        b.broadcast(this.getLastTouchPosition(), e);
-      }
+      var touches = e.changedTouches;
+      this._lastTouchPositions = [];
+      var ids: number[] = [];
+      for (var i = 0; i < touches.length; i++) {
+        var touch = touches[i];
+        var touchID = touch.identifier;
+        var newTouchPosition = this.translator.computePosition(touch.clientX, touch.clientY);
+        if (newTouchPosition != null) {
+          ids.push(touchID);
+          this._lastTouchPositions[touchID] = newTouchPosition;
```

Comment:
I can move the array to be more of an object model if you want.  The idea is similar to a hashmap